ur_robot_driver/src/ros/hardware_interface.cpp

```diff
@@ -382,6 +389,128 @@ bool HardwareInterface::init(ros::NodeHandle& root_nh, ros::NodeHandle& robot_hw
   return true;
 }
 
+void HardwareInterface::registerJointLimits(ros::NodeHandle& robot_hw_nh,
+                                            const hardware_interface::JointHandle& joint_handle_position,
+                                            const hardware_interface::JointHandle& joint_handle_velocity,
+                                            std::size_t joint_id)
+{
+  // Default values
+  joint_position_lower_limits_[joint_id] = -std::numeric_limits<double>::max();
```

Comment:
As far as I can see, this vector doesn't get resized anywhere, which would making this call illegal.